help.md

```diff
@@ -0,0 +1,901 @@
+    can be used in other applications, analyzed beyond the capability of the
+    ATT&CK Navigator, and generated by tools for import into the Navigator.
+    The Layer file format is described <a href="assets/NavigatorLayerFileFormatv4_1.pdf">here</a>.
+</p>
+
+
+<span class="creating_new_layers"></span>
+## Creating New Layers
+
+<p>
+    To create a new layer, open a new tab and click the "Create New Layer" dropdown.
+    The quick access buttons will create a layer with the current version of ATT&CK.
+    Past versions of ATT&CK compatible with the Navigator are accessible in the
+    "More Options" dropdown. This interface allows you to specify the desired version
+    and domain for the new layer. Note that for performance reasons, the Navigator
+    currently imposes a limit of ten (10) active layers at any given point in time.
+</p>
+
+
+<span class="saving_and_loading_layers"></span>
+## Saving and Loading Layers
+
+<p>
+    Layers can be saved by clicking the "save layer" button (<img src="assets/icons/ic_file_download_black_24px.svg">). This will open a dialog
+    to save a layer configuration file to your local computer. This contains
+    the configuration for the techniques that have been customized
+    (commented, colored, assigned a score, or disabled) as well as the
+    scoring gradient setup, filter selection, layer name, layer description,
+    view configuration.
+</p><br>
+<p>
+    Saved layer configuration files can be opened in the ATT&CK Navigator to
+    restore a layer you've worked on previously. To do so, open a new tab
+    and open the "Open Existing Layer" panel. Then click "Upload from
+    local", and select your saved configuration file. Doing so will restore
+    your saved layer to the ATT&CK Navigator. This interface also has a "load from URL"
+    input allowing you to open a layer json from a remote source.
+</p>
+
+
+<span class="creating_layers_from_other_layers"></span>
+## Creating Layers from Other Layers
+
+<p>
+    Layers can be created which inherit properties from other layers. Several fields
+    exist which can be used to choose which layers to inherit properties from:
+</p>
+<ul>
+    <li>
+        <p>
+            <b>Domain:</b> choose the domain and version for the new layer. Layers can only inherit
+            properties from other layers of the same domain and version.
+        </p>
+    </li>
+    <li>
+        <p>
+            <b>Score Expression:</b> Technique
+            <span class="section-link linkName-Scoring_Techniques">scores</span> in the created layer are
+            initialized to the result of this expression. This field should take
+            the form of an equation or constant, using variables for layers.
+            Layer variables are shown in yellow on tabs when the "create layer
+            from other layers" panel is opened:
+        </p>
+        <img src="assets/image_scoreVariableExample.png" height="100px">
```

Comment:
Rendered on GitHub the images do not work: 

- Shows as blank/broken image:
   ![image](https://user-images.githubusercontent.com/28868888/124280756-118b3a80-db17-11eb-9d86-66a823263aa4.png)
- Or as their alt text if alt text is specified:
   ![image](https://user-images.githubusercontent.com/28868888/124280711-033d1e80-db17-11eb-845e-36312aff237f.png)

This is because the images point to the assets folder but are missing part of the path for when opened on GitHub. The path is correct for when they're rendered in-app. When loaded in GitHub this is an example URL:
```
/mitre-attack/attack-navigator/raw/feature/%23179-help-page/assets/icons/ic_sort_alphabetically_ascending_black_24px.svg
```

The correct URL would be:
```
/mitre-attack/attack-navigator/raw/feature/%23179-help-page/nav-app/src/assets/icons/ic_sort_alphabetically_ascending_black_24px.svg
```
So it's missing the `/nav-app/src` part of the path for being rendered in GitHub.

In order to be displayed in both places properly you'll probably need to make the in-app version change the path to remove the `/nav-app/src` prefix which will be required for the GitHub view. I'm pretty sure u[pdating the renderer](https://github.com/jfcere/ngx-markdown#renderer0 for `img` tokens would be a workable approach for this.